pkg/proxy/ipvs/proxier.go

```diff
@@ -856,21 +856,25 @@ func CleanupLeftovers(ipvs utilipvs.Interface, ipt utiliptables.Interface, ipset
 			encounteredError = true
 		}
 	}
+
 	// Delete dummy interface created by ipvs Proxier.
 	nl := NewNetLinkHandle()
-	err := nl.DeleteDummyDevice(DefaultDummyDevice)
-	if err != nil {
-		glog.Errorf("Error deleting dummy device %s created by IPVS proxier: %v", DefaultDummyDevice, err)
-		encounteredError = true
+	if found, _ := nl.IsDeviceExist(DefaultDummyDevice); found {
```

Comment:
why not log error here? tell people dummy device doesn't exist thus we can't delete it.

Reply:
Done, thanks